service/api_task.go

```diff
@@ -143,6 +143,14 @@ func (as *APIServer) EndTask(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	now := time.Now()
+	grip.Info(message.Fields{
```

Comment:
It seems unnecessary to log immediately

Reply:
True, but I was slightly interested in JSON parsing time.